src/components/Tables/components/ConfirmDelete.tsx

```diff
@@ -0,0 +1,41 @@
+import { AlertDialog, Button, Flex } from '@radix-ui/themes'
+
+type ConfirmDeleteProps = {
+    open: boolean
+    setOpen: (open: boolean) => void
+    onDelete: () => void
+}
+
+export const ConfirmDelete: React.FC<ConfirmDeleteProps> = ({
```

Comment:
Why type this here? Please just call the props type `Props`. And remove this typing as we don't use this anywhere else in the codebase.